Pomegranate.prefab

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Move this file to [assets/prefabs/food/raw](https://github.com/Terasology/Cooking/tree/master/assets/prefabs/food/raw) where all other fruits are.